examples/quota-management/README.md

```diff
@@ -1,34 +1,47 @@
-# Quota Management example
+# Managing the device's quota data with the emnify Java SDK
 
-This example shows how to manage endpoint's quota data.
+This example shows how to manage the device's quota data.
 
-1. Script authorizes the client using configured credentials.
-2. The list of endpoints is fetched then a first endpoint is taken to work with.
-3. Then check the current state of quota
-4. Script configures a quota with the following data:
-    - set new volume of 10MB
-    - the quota expires in 7 days
-    - a threshold is set to 15%. If the remaining volume goes below this percentage of the applied quota, the system
-      will generate a "threshold reached" event.
-    - the action of exhaustion is configured to Throttle with a peak throughput of 64000 octets per second
-5. Script fetches the quota data to check if configuration has been saved successfully
+Here's a high-level overview of what happens in the script:
+
+1. Authorizes the client using configured credentials
+1. Fetches a list of devices and selects the first device
+1. Checks the current state of the quota
+1. Configures a new quota with the following data:
```

Comment:
For a configuration, "data" just seems too general.

Any one of the following would be preferable:
- settings
- values
- parameters
- attributes
- constraints